### DISCUSSION TITLE
Welcome to encfs4win Discussions!

### DISCUSSION THREAD

**Discussion by jetwhiz** [jetwhiz/encfs4win#150] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amicalmant** [comment]
Merci beaucoup for the good work, Charles.

As you can see in my profile, I am a Web writer, webmaster, webcaster & web video producer based in Rimouski, Quebec (Canada). I am not a developer but I support and may contribute to open source projects in several ways: translation (EN/FR), communication (FR/EN), and small donations (CAD😄).

I have been playing with the application layer of web servers since 1997 and got involved one way or another with open source software since 1998. I was able to switch my main personal computer to Linux in 2011 only and I love it that way. However, editing video is a bit shaky on Linux so I keep Windows partitions ready and use them quite regularly too. This is why this project is so useful to me.
